config/initializers/hyrax.rb

```diff
@@ -279,3 +279,9 @@
   path = file_location_uri.sub(/^#{Hyrax.config.derivatives_path}/, '')
   "#{Settings.streaming.base_url}#{path}/index.m3u8"
 end
+
+require 'hyrax/search_state'
+Hyrax::Admin::AdminSetsController.admin_set_create_service = Hyrax::AvalonAdminSetCreateService
+AdminSet.class_eval do
```

Comment:
I think this could be done without class_eval, but using prepend instead:
`AdminSet.prepend AdminSetDropbox`
I'm basing this on https://github.com/samvera/hyrax/pull/1041#issuecomment-303918649

Reply:
I've made the changes, except prepend which didnt work.